packages/web/src/pages/sign-up-page/components/SocialMediaLoginOptions.tsx

```diff
@@ -74,28 +71,24 @@ export const SocialMediaLoginOptions = ({
       ) : null}
       {isInstagramEnabled ? (
         <SignupFlowInstagramAuth
-          className={styles.flex1}
+          css={{ flex: 1 }}
           onFailure={handleFailure}
-          onSuccess={({ handle, requiresReview }) =>
-            handleSuccess({ handle, requiresReview, platform: 'instagram' })
-          }
+          onSuccess={handleSuccess}
         >
           <SocialButton
             type='button'
             fullWidth
             socialType='instagram'
-            className={styles.flex1}
+            css={{ flex: 1 }}
             aria-label={messages.signUpInstagram}
           />
         </SignupFlowInstagramAuth>
       ) : null}
       {isTikTokEnabled ? (
-        <Box className={styles.flex1}>
+        <Box css={{ flex: 1 }}>
```

Comment:
wonder if we should add this to harmony since we use it a lot

Reply:
or is it a reasonable default for Box? 🤔 That sounds a bit scary

Reply:
we should probably add it to flex at least yeah?